packages/cloudfront-signer/src/sign.ts

```diff
@@ -327,14 +343,20 @@ class CloudfrontSignBuilder {
     };
   }
 
-  private signData(data: string, privateKey: string | Buffer): string {
+  private signData(data: string, privateKey: string | Buffer, passphrase: string): string {
     const sign = createSign("RSA-SHA1");
     sign.update(data);
-    return sign.sign(privateKey, "base64");
+    return sign.sign(
+      {
+        key: privateKey,
+        passphrase: passphrase,
+      },
+      "base64"
+    );
```

Comment:
nit: can be shortened
```suggestion
    return sign.sign({ key: privateKey, passphrase }, "base64");
```

The passphrase is optional in PrivateKeyInput, so we don't have to do undefined check

Reply:
Oh, thanks for your recommendation.